docker/build_jakartaeetck.sh

```diff
@@ -18,7 +18,6 @@ if [ -z "$ANT_HOME" ]; then
   export ANT_HOME=/usr/share/ant/
 fi
 
```

Comment:
@gurunrao 

I think that we should always build with `JDK11` for EE 10.  With this pull request, we should only be changing the running of TCK tests to be able to use either `JDK11` or `JDK17`.

Reply:
With update to docker image "jakartaee/cts-base:0.3" default JAVA_HOME is set to "/opt/jdk-11.0.2"(replacing jdk8 java_home), hence we need not set JAVA_HOME for build.